src/mapping.ts

```diff
@@ -51,7 +51,7 @@ export function handleTrade(event: Trade): void {
   let buyAmount = event.params.buyAmount
   let network = dataSource.network()
 
-  let timestamp = event.block.timestamp
+  let timestamp =  event.block.timestamp.toI32()
```

Comment:
Nitpick: unnecessary whitespace

```suggestion
  let timestamp = event.block.timestamp.toI32()
```